management_api_app/api/routes/workspaces.py

```diff
@@ -45,14 +51,16 @@ async def create_workspace(workspace_create: WorkspaceInCreate, workspace_repo:
         workspace_repo.save_workspace(workspace)
     except Exception as e:
         logging.error(f"Failed to save workspace instance in DB: {e}")
-        raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE, detail=strings.STATE_STORE_ENDPOINT_NOT_RESPONDING)
+        raise HTTPException(status_code=status.HTTP_503_SERVICE_UNAVAILABLE,
```

Comment:
This only looks like a formatting change but we are not following the 80 char rule in this repo, can we return this to be an n the original state? Same for other similar formatting changes - for consistency